login.html

```diff
@@ -1,45 +1,49 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Apni-Dukaan</title>
-  <link rel="stylesheet" type="text/css" href="./static/login.css">
-  <script src="./static/login.js"></script>
-  <link rel="icon" href="./Images/favicon.ico" type="image/x-icon">     
-  </head>    
+	<title>Animated Login Form</title>
+	<link rel="stylesheet" type="text/css" href="static/login.css">
+	<link href="https://fonts.googleapis.com/css?family=Poppins:600&display=swap" rel="stylesheet">
+	<script src="https://kit.fontawesome.com/a81368914c.js"></script>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
 <body>
-  <div class="login_form">
-    <div class="details">
-      <div class="welcome">Apni Dukaan</div>
-      <form>
-        <div class="wrap">
-          <label>Email</label>
-          <input type="text" class="input" id="email">
-          </div>
-        <div class="wrap">
-          <label>Password</label>
-          <input type="password" class="input" data-type="password" id="pwd1">
-          </div>
-        <div class="wrap">
-          <a href="forgotPassword.html" title="Forgot Password" id="link-reset">Forgot Password?</a>
-          </div>
-        <button class="button" value="Login" onclick="return login()"><h1 class="sign">Sign in!</h1></button>
-      </form>
+	<img class="wave" src="img/wave.png">
+	<div class="container">
+		<div class="img">
+			<img src="img/bg.svg">
+		</div>
+		<div class="login-content">
+			<form action="index.html">
+				<img src="img/Logo.png">
+				<h2 class="title">Welcome Back!</h2>
+        <p font-size="17px">Log in and use the opportunities offered by our portal.</p>
+           		<div class="input-div one">
+           		   <div class="i">
+           		   		<i class="fas fa-user"></i>
+           		   </div>
+           		   <div class="div">
+           		   		<h5>Email</h5>
+           		   		<input type="Email" class="input">
+           		   </div>
+           		</div>
+           		<div class="input-div pass">
+           		   <div class="i"> 
+           		    	<i class="fas fa-lock"></i>
+           		   </div>
+           		   <div class="div">
+           		    	<h5>Password</h5>
+           		    	<input type="password" class="input">
+            	   </div>
+            	</div>
+            	<a href="#">Forgot Password?</a>
+            	<input type="submit" class="btn" value="Login">
+              <p>Don't have an account</p><a href="https://qazimaazarshad.github.io/Apni-Dukaan/signup.html">Sign Up</a>
```

Comment:
Don't use links like this. 
Redirect it to the signup page using this way "/folderfilelocation".

Update links everywhere where you have done this.